release.json

```diff
@@ -1,22 +1,22 @@
 {
     "nightly": {
-        "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "master",
-        "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+        "INTEGRATIONS_CORE_VERSION": "7.29.x",
+        "OMNIBUS_SOFTWARE_VERSION": "7.29.x",
+        "OMNIBUS_RUBY_VERSION": "7.29.x",
         "JMXFETCH_VERSION": "0.44.1",
         "JMXFETCH_HASH": "fd369da4fd24d18dabd7b33abcaac825d386b9558e70f1c621d797faec2a657c",
-        "MACOS_BUILD_VERSION": "master",
+        "MACOS_BUILD_VERSION": "7.29.x",
         "WINDOWS_DDNPM_DRIVER": "release-signed",
         "WINDOWS_DDNPM_VERSION": "0.98.2.git.86.53d1ee4",
         "WINDOWS_DDNPM_SHASUM": "5d31cbf7aea921edd5ba34baf074e496749265a80468b65a034d3796558a909e"
     },
     "nightly-a7": {
-        "INTEGRATIONS_CORE_VERSION": "master",
-        "OMNIBUS_SOFTWARE_VERSION": "master",
-        "OMNIBUS_RUBY_VERSION": "datadog-5.5.0",
+        "INTEGRATIONS_CORE_VERSION": "7.29.x",
+        "OMNIBUS_SOFTWARE_VERSION": "7.29.x",
+        "OMNIBUS_RUBY_VERSION": "7.29.x",
         "JMXFETCH_VERSION": "0.44.1",
         "JMXFETCH_HASH": "fd369da4fd24d18dabd7b33abcaac825d386b9558e70f1c621d797faec2a657c",
-        "MACOS_BUILD_VERSION": "master",
+        "MACOS_BUILD_VERSION": "7.29.x",
```

Comment:
Should `SECURITY_AGENT_POLICIES_VERSION` also target `7.29.x`?

Note: There is no `SECURITY_AGENT_POLICIES_VERSION` in `nightly` section. Is it expected?

Reply:
They have never branched out, so I think this is expected, I have asked them to clarify.